test/test.py

```diff
@@ -455,11 +350,12 @@ def jacobian_fd_approx(mass_fractions, delta_y):
      ("sandiego", "H2", 0.5),
      ("hong", "H2", 0.5)]
 )
-def test_falloff_kinetics(mechname, fuel, stoich_ratio):
+@pytest.mark.parametrize("user_np", numpy_list)
+def test_falloff_kinetics(mechname, fuel, stoich_ratio, user_np, request):
     """This function tests that pyrometheus-generated code
     computes the Cantera-predicted falloff rate coefficients"""
-    sol = ct.Solution(f"mechs/{mechname}.yaml", "gas")
-    ptk = pyro.codegen.python.get_thermochem_class(sol)()
+
+    sol, ptk = pyro_init(mechname, user_np, request)
```

Comment:
That this routine returns a `ct.Solution` is somewhat problematic to me. It initializes pyro (whatever that means), which uses a `ct.Solution` as one of its inputs. So, `sol` is not a product of Pyro initialization.

Reply:
See my other comment `pyro_init`.

Reply:
I'm curious as to whether the Pyro object can carry the `sol` as a member.

Reply:
I responded to this in another thread. I thought this change would be cumbersome and that the current approach isn't a _sin_ per se.

Reply:
Cumbersome indeed.